e2e/cypress/integration/channel_sidebar/more_unreads_position_with_scroll_spec.js

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @channel_sidebar
+
+import {getRandomId} from '../../utils';
+
+describe('Channel sidebar', () => {
+    let mainUser;
+    let otherUser;
+    let myTeam;
+
+    // This will ensure a list of channel names that will be alphabetically sorted
+    const channelNames = new Array(20).fill(1).map((value, index) => `scroll${index}`);
+
+    before(() => {
+        cy.apiInitSetup().then(({team, user: firstUser}) => {
+            mainUser = firstUser;
+            myTeam = team;
+
+            cy.apiCreateUser().then(({user: secondUser}) => {
+                otherUser = secondUser;
+                return secondUser.id;
+            }).then((userId) => cy.apiAddUserToTeam(team.id, userId));
+
+            cy.wrap(channelNames).each((name) => {
```

Comment:
This seems to be the recommended way to make a proper loop with Cypress. Wrap a static array and then use the `cy.each` over each item.

Reply:
Cypress includes lodash. Another way is to do it like:
```javascript
Cypress._.times(20, (num) => {
    const name = `channel-scroll${num}`;
    ...
});
```

Reply:
On second thought, after trying that option, I do re-use my generated array of names a few times in the test to fetch specific channels (first, and last) and scroll into view.
I think it would be best to keep it the way it is, otherwise I will have to hard-code a lot of things.

Are you OK with it ?

Reply:
Yeah, I've seen it and I thought it's okay to hard-code the first and last channel names but I'm good as is.  Thanks for trying :)